tslint.json

```diff
@@ -0,0 +1,12 @@
+{
+	"defaultSeverity": "error",
+	"extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier"],
+	"jsRules": {},
+	"rulesDirectory": ["tslint-plugin-prettier"],
+	"rules": {
+		"prettier": true
```

Comment:
add `"member-access": false` to prevent us from facing with error `' The class method ‘render’ must be marked either ‘private’, ‘public’, or ‘protected’’`